src/ClimaLSM.jl

```diff
@@ -222,65 +209,60 @@ function prognostic_vars(land::AbstractLandModel)
     prognostic_list = map(components) do model
         prognostic_vars(getproperty(land, model))
     end
-    return NamedTuple{components}(prognostic_list)
+    return (NamedTuple{components}(prognostic_list),)
 end
 
 function prognostic_types(land::AbstractLandModel)
     components = land_components(land)
     prognostic_list = map(components) do model
         prognostic_types(getproperty(land, model))
     end
-    return NamedTuple{components}(prognostic_list)
+    return (NamedTuple{components}(prognostic_list),)
 end
 
+
 function auxiliary_vars(land::AbstractLandModel)
     components = land_components(land)
     auxiliary_list = map(components) do model
         auxiliary_vars(getproperty(land, model))
     end
-    return NamedTuple{(components..., :interactions)}((
-        auxiliary_list...,
-        interaction_vars(land),
-    ))
+    return (add_lsm_aux_vars(land)..., NamedTuple{components}(auxiliary_list))
 end
 
 function auxiliary_types(land::AbstractLandModel)
     components = land_components(land)
     auxiliary_list = map(components) do model
         auxiliary_types(getproperty(land, model))
     end
-    return NamedTuple{(components..., :interactions)}((
```

Comment:
We dont need to change this, but if we keep it as a NamedTuple, we need to give a key for the additional LSM aux vars. since they arent stored in the cache with a key, this feels misleading,so I changed to a tuple. I also changed the prognostic methods in the same way.